tests/git-log.spec.js

```diff
@@ -183,6 +183,7 @@ describe('GitLog', function() {
             'Currently we had AppImage scripts and other resources in various',
             'different places in the code base.'
           ].join('\n'),
+          hash: undefined,
```

Comment:
Even better: why not updating the `utils.formatCommit` function to also accept a hash, and check that is propagated correctly?